docs/user/addons.rst

```diff
@@ -0,0 +1,53 @@
+Read the Docs Addons
+====================
+
+**Read the Docs Addons** is a group of features for documentation readers and maintainers that you can add to any documentation set hosted on Read the Docs:
+
+:doc:`Traffic analytics </analytics>`
+    explore pageviews for your docs
+
+:doc:`DocDiff </pull-requests>`
+    highlight changed output from pull requests
+
+:doc:`Pull request notification </pull-requests>`
+    notify readers that they are reading docs from a pull request
+
+:doc:`Flyout </flyout-menu>`
+    easily switch between versions and translations
+
+:doc:`Non-stable notification </notifications>`
+    notify readers that they are reading docs from a non stable release
+
+:doc:`Latest version notification </notifications>`
+    notify readers that they are reading docs from an older version
```

Comment:
I just realized that since we are linking to _existing pages_ for these addons, both of these notifications would be related to `/versions` page (https://docs.readthedocs.com/en/stable/versions.html).

So, it may make sense to move the content from the notifications page into its own section in the versions page. Actually, we already have a section for the "Version warning" there: https://docs.readthedocs.com/en/stable/versions.html#version-warning. We can adapt it/refactor with the new content.

I'm sorry I'm mentioning this at this stage in the review, but I didn't realized about this earlier 😓